### ISSUE TITLE
arcface backbone to tensorrt

### ISSUE THREAD

**Issue by guangdongliang** [wujiyang/Face_Pytorch#35]
Thank for your work!
       I want to convert arcface to tensorrt, but [nn.BatchNorm1d(feature_dim))](https://github.com/wujiyang/Face_Pytorch/blob/master/backbone/arcfacenet.py#L146) only do not support 2D in tensorrt. the error message is what():  No converter registered for op type: Squeeze. 
       In torch/onnx/symbolic.py, 2D tensor will be unsqueeze to 3D first and then squeeze to 2D.

look forward to your replay! thank you!
